src/js/controllers/imageCtrl.js

```diff
@@ -0,0 +1,35 @@
+(function (angular) {
+    angular
+        .module('flickrPOC')
+        .controller('imagesCtrl', imagesCtrl);
+
+    imagesCtrl.$inject = ['images'];
+
+    function imagesCtrl(images) {
+        /* jshint validthis: true */
+        var vm = this;
```

Comment:
I understand why the `validthis` rule needs to be disabled. JSHint doesn't like the reference to `this`, and although the general override is effective, it disables the rule for all lines that follow it. This can cause issues where invalid code isn't discovered by the linter. What is preferred is simply disabling the rule on a line-by-line basis, as required. 

So, this:
```javascript
/* jshint validthis: true */
var vm = this;
```
Becomes this:
```javascript
var vm = this; // jshint ignore:line
```
The single line ignore allows other developers to know exact what line is causing the linter to complain, thus can feel confident that the rest of the file is tip-top.